components/fileselect/validation.md

```diff
@@ -8,43 +8,68 @@ published: true
 position: 10
 ---
 
-# FileSelect - Selected Files Validation
+# FileSelect File Validation
 
 If you want to validate the selected files, you should implement the validation in two parts:
 
-* client validation - performed by the Telerik FileSelect component
-* server validation - must be implemented in the application endpoints
+* Client validation&mdash;performed by the Telerik FileSelect component
+* Server validation&mdash;must be implemented in the application backend or endpoints
 
-The Telerik FileSelect component offers parameters to validate the file selection on the client:
+## Parameters
 
-* `Accept` - `string` - not validation per se, but this parameter can [instruct the browser what file types to allow users to select](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/accept).
-* `AllowedExtensions` - `List<string>` - a list of valid file extensions. Choosing other file types will mark them as invalid in the UI. The default value is `null`, which allows all extensions.
-* `MinFileSize`- `int?` - the minimum size of a file in bytes. Files with a smaller size will be marked as invalid in the UI.
-* `MaxFileSize`- `int?` - the maximum size of a file in bytes. Files with a larger size will be marked as invalid in the UI.
+The Telerik [FileSelect component offers parameters](slug:Telerik.Blazor.Components.TelerikFileSelect) to validate the file selection on the client:
 
->caption Client validation in the Telerik FileSelect component
+* `AllowedExtensions`
+* `MinFileSize`
+* `MaxFileSize`
 
-For brevity, this sample does not showcase actual upload of the files. You can find an example in the [FileSelect Events article](slug:fileselect-events).
+Selected files that don't meet the defined criteria are marked as invalid in the component UI.
+
+The FileSelect also provides and `Accept` parameter. It does not enforce validation, but [instructs the browser what file types to allow users to select](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/accept). This can also help users find the correct files more easily.
```

Comment:
```suggestion
The FileSelect also provides an `Accept` parameter. It does not enforce validation, but [instructs the browser what file types to allow users to select](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/accept). This can also help users find the correct files more easily.
```